ui/src/assets/i18n/en.json

```diff
@@ -248,7 +248,10 @@
     "darkTheme": "dark",
     "defaultTheme": "default",
     "results": "Results",
-    "resultsDescription": "You can choose the number of results per page."
+    "resultsDescription": "You can choose the number of results per page.",
+    "reverseProxyFix": "Reverse-Proxy-Fix",
+    "reverseProxyFixActive": "Activate",
+    "reverseProxyFixDescription": "You can activate the fix for the reverse proxy operation."
```

Comment:
Hi @klopfdreh , we are going to merge this in for the upcoming `2.11.5` release. Could you adjust the message here a bit and remove the term "fix", something like:
```
You can activate the client-side reverse proxy feature."
```

Reply:
Sure, I adjusted the other languages as well and also changed the label in front of the description.
```json
    "reverseProxyFix": "Reverse-Proxy-Feature",
    "reverseProxyFixActive": "Activate",
    "reverseProxyFixDescription": "You can activate the client-side reverse proxy feature."
```
```json
    "reverseProxyFix": "Reverse-Proxy-Feature",
    "reverseProxyFixActive": "Activate",
    "reverseProxyFixDescription": "You can activate the client-side reverse proxy feature."
```
```json
    "reverseProxyFix": "Функция обратного прокси",
    "reverseProxyFixActive": "Активировать",
    "reverseProxyFixDescription": "Вы можете активировать функцию обратного прокси-сервера на стороне клиента."
```
<img width="857" alt="Bildschirmfoto 2024-09-12 um 18 30 37" src="https://github.com/user-attachments/assets/b70d6e69-5208-43a6-b21d-6b5d549d15a1">
<img width="777" alt="Bildschirmfoto 2024-09-12 um 18 33 02" src="https://github.com/user-attachments/assets/ab30b34d-c04e-4117-918e-e6354cadee51">

I keep the naming of the variables in the code as `*fix`

Reply:
Thanks @klopfdreh .